Tests/UnitTests/Core/Vertexing/AdaptiveMultiVertexFinderTests.cpp

```diff
@@ -94,7 +97,8 @@ BOOST_AUTO_TEST_CASE(adaptive_multi_vertex_finder_test) {
   Finder finder(finderConfig);
   Finder::State state;
 
-  auto tracks = getAthenaTracks();
+  auto csvData = readTracksAndVertexCSV(DATAPATH);
+  auto tracks = std::get<2>(csvData);
```

Comment:
you mind making the index an enum?

Reply:
Done. Pls check if that's what you had in mind.